src/Nncase.Importer/Onnx/Reduce.cs

```diff
@@ -51,7 +51,7 @@ private Expr ReduceCore(in NodeProto op, ReduceOp reduceOp, float initValue, Fun
                 var x when x == ReduceOp.Max && input.CheckedDataType == DataTypes.Int32 => F.Tensors.Reduce(reduceOp, f(input), axis, int.MinValue, keepDims),
                 var x when x == ReduceOp.Min && input.CheckedDataType == DataTypes.Int64 => F.Tensors.Reduce(reduceOp, f(input), axis, long.MaxValue, keepDims),
                 var x when x == ReduceOp.Min && input.CheckedDataType == DataTypes.Int32 => F.Tensors.Reduce(reduceOp, f(input), axis, int.MaxValue, keepDims),
-                _ => F.Tensors.Reduce(reduceOp, f(input), axis, F.Tensors.Cast(initValue, input.CheckedDataType), keepDims),
+                _ => F.Tensors.Reduce(reduceOp, f(input), axis, F.Tensors.Cast(initValue, input.CheckedDataType, CastMode.CheckOverflow), keepDims),
```

Comment:
Should add bfloat16 and float16 reduce cases in pytest.

Reply:
done